code/modules/events/wizard/curseditems.dm

```diff
@@ -11,51 +24,72 @@
 //item you want to equip to the hand, and set its slots_flags = null. Only items equiped to hands need do this.
 
 /datum/round_event/wizard/cursed_items/start()
-	var/item_set = pick("wizardmimic", "swords", "bigfatdoobie", "boxing", "voicemodulators", "catgirls2015")
+	var/item_set = pick(
+		BIG_FAT_DOOBIE,
+		BOXING,
+		CATGIRLS_2015,
+		CURSED_SWORDS,
+		VOICE_MODULATORS,
+		WIZARD_MIMICRY,
+	)
 	var/list/loadout[SLOTS_AMT]
-	var/ruins_spaceworthiness
-	var/ruins_wizard_loadout
+	var/ruins_spaceworthiness = FALSE
+	var/ruins_wizard_loadout = FALSE
 
 	switch(item_set)
-		if("wizardmimic")
-			loadout[ITEM_SLOT_OCLOTHING] = /obj/item/clothing/suit/wizrobe
-			loadout[ITEM_SLOT_FEET] = /obj/item/clothing/shoes/sandal/magic
-			loadout[ITEM_SLOT_HEAD] = /obj/item/clothing/head/wizard
-			ruins_spaceworthiness = 1
-		if("swords")
-			loadout[ITEM_SLOT_HANDS] = /obj/item/katana/cursed
-		if("bigfatdoobie")
+		if(BIG_FAT_DOOBIE)
 			loadout[ITEM_SLOT_MASK] = /obj/item/clothing/mask/cigarette/rollie/trippy
-			ruins_spaceworthiness = 1
-		if("boxing")
+			ruins_spaceworthiness = TRUE
+		if(BOXING)
 			loadout[ITEM_SLOT_MASK] = /obj/item/clothing/mask/luchador
 			loadout[ITEM_SLOT_GLOVES] = /obj/item/clothing/gloves/boxing
-			ruins_spaceworthiness = 1
-		if("voicemodulators")
-			loadout[ITEM_SLOT_MASK] = /obj/item/clothing/mask/chameleon
-		if("catgirls2015")
+			ruins_spaceworthiness = TRUE
+		if(CATGIRLS_2015)
 			loadout[ITEM_SLOT_HEAD] = /obj/item/clothing/head/costume/kitty
-			ruins_spaceworthiness = 1
-			ruins_wizard_loadout = 1
+			ruins_spaceworthiness = TRUE
+			ruins_wizard_loadout = TRUE
+		if(CURSED_SWORDS)
+			loadout[ITEM_SLOT_HANDS] = /obj/item/katana/cursed
+		if(VOICE_MODULATORS)
+			loadout[ITEM_SLOT_MASK] = /obj/item/clothing/mask/chameleon
+		if(WIZARD_MIMICRY)
+			loadout[ITEM_SLOT_OCLOTHING] = /obj/item/clothing/suit/wizrobe
+			loadout[ITEM_SLOT_FEET] = /obj/item/clothing/shoes/sandal/magic
+			loadout[ITEM_SLOT_HEAD] = /obj/item/clothing/head/wizard
+			ruins_spaceworthiness = TRUE
 
-	for(var/mob/living/carbon/human/H in GLOB.alive_mob_list)
-		if(ruins_spaceworthiness && !is_station_level(H.z) || isspaceturf(H.loc) || isplasmaman(H))
+	var/list/mob/living/carbon/human/victims = list()
+
+	for(var/mob/living/carbon/human/target in GLOB.alive_mob_list)
+		if((ruins_spaceworthiness && !is_station_level(target.z)) || isspaceturf(target.loc) || isplasmaman(target))
```

Comment:
it should check the species for if it requires a special outfit (its a var i tihnk) instead of a plasmaman check